packages/datagrid/src/components/PinHeaderRenderer/PinHeaderRenderer.component.tsx

```diff
@@ -1,8 +1,6 @@
-import React from 'react';
-import { ButtonIcon } from '@talend/design-system';
 import theme from './PinHeaderRenderer.scss';
-
-export const PIN_HEADER_RENDERER_COMPONENT = 'pinHeaderRenderer';
+import { ButtonIcon } from '@talend/design-system';
+import React from 'react';
```

Comment:
Shouldn't the linter ask for this import to be first? Or did I imagine that 🤔

Reply:
linter goes brrr